### PR TITLE
CHE-2178 fix copy/paste IDE-terminal actions for Firefox browser

### PR DESCRIPTION
**Pull request by olexii4**
According to the document: https://www.w3.org/TR/clipboard-apis/#paste-event
''The paste action has no effect in a non-editable context, but the event fires regardless."
But we haven't copy/paste events in firefox if we in a non-editable context. That is why we add invisible editable context add focus on it before copy/past actions (it is only for firefox).

#2178

Signed-off-by: Oleksii Orel <oorel@codenvy.com>
@ashumilova  @akurinnoy

### CODE REVIEW

File: websocket-terminal/src/main/resources/term.js [hunk 1]
```diff
@@ -579,11 +633,19 @@ Terminal.bindCopy = function(document) {
...
-    if (!term._selected) return;
     var textarea = term.getCopyTextarea();
-    var text = term.grabText(
-      term._selected.x1, term._selected.x2,
-      term._selected.y1, term._selected.y2);
+    var text;
+    if (term.isFirefox) {
+      text = Terminal.textSelectionBuffer;
+      Terminal.clearSelection();
+        ev.clipboardData.setData('text/plain', text);
```
Comment (benoitf) [inline websocket-terminal/src/main/resources/term.js]: alignment ?

**Comment by skabashnyuk** [comment]
Ok if @benoitf  is ok

File: websocket-terminal/src/main/resources/term.js (hunk 1 above)
Comment (olexii4) [inline websocket-terminal/src/main/resources/term.js]: ok

**Comment by benoitf** [comment]
@olexii4 tested and it works great on Firefox and it seems it works better as well on Chrome on Mac than before :-)

**Pull request merged**